html_css/v2/main-course/more_css/more-css.md

```diff
@@ -0,0 +1,53 @@
+### Introduction
```

Comment:
What do you think about renaming this lesson to something like `enhancing-design` / `enhancing-design-with-css` or something similar. I think it would be a little more descriptive of what students will get from this lesson.

Reply:
yep.  dig it